client/settings/express-checkout-settings/general-payment-request-button-settings.js

```diff
@@ -206,6 +206,7 @@ const GeneralPaymentRequestButtonSettings = ( { type } ) => {
 				value={ buttonType }
 				options={ buttonActionOptions }
 				onChange={ setButtonType }
+				__nextHasNoMarginBottom
```

Comment:
36px default size for wp.components.SelectControl is deprecated since version 6.8 and will be removed in version 7.1.
```suggestion
				__nextHasNoMarginBottom
				__next40pxDefaultSize
```